tests/tests/relayer.rs

```diff
@@ -143,6 +141,79 @@ async fn relayer_can_download_logs() {
     eth_node_handle.shutdown.send(()).unwrap();
 }
 
+#[tokio::test(flavor = "multi_thread")]
```

Comment:
It would be nice to have a test when two endpoints return the different result for the same request

Reply:
I just added it in the new tests I just made :)